tests/inkscape-with-config.expected

```diff
@@ -0,0 +1,28 @@
+
+  [32mAn[0m SVG (Scalable Vector Graphics) editing program.
+  [32mUse[0m -z to not open the GUI and only process files in the console.
```

Comment:
Hm, I still find it strange that `An` and `Use` are highlighted. Could we avoid that?

Reply:
The idea was to visually distinguish the description from example text, to allow it to be found with a quick glance due to its high contrast unique pattern. I'm open to alternatives, not separating it from example text would seems like a missed opportunity.

Reply:
Well, it's the first thing after the headline. And it's not indented. Adding a configurable style for it would be OK, but highlighting the first word seems confusing to me :)

Reply:
It already has a configurable style, however it is indented on the same level as the example text, still I agree that the split word highlighting may be confusing and could collide with a custom style. So I'll take it out.